.travis.yml

```diff
@@ -12,5 +12,15 @@ notifications:
   email: false
 
 script:
-  - yarn lint
-  - yarn danger ci
+- yarn lint
+- yarn danger ci
+
+deploy:
+  provider: npm
+  email: xander.marjoram@just-eat.com
```

Comment:
is there a way you can change this to ui@just-eat.com in the future so it isn't tied to your email?

Reply:
@DamianMullins I'm happy to change it now if I can also get a token from that email address

Reply:
Is this a token that you get through Travis?